packages/testing/package.json

```diff
@@ -50,8 +50,8 @@
   },
   "devDependencies": {
     "@types/jest": "^29.2.0",
-    "@types/node": "^14.6.1",
-    "@types/node-fetch": "^2.5.4",
+    "@types/node": "^18.11.18",
+    "@types/node-fetch": "^2.6.2",
```

Comment:
I guess we should drop `node-fetch` and its typing here too.

Reply:
It was added back in https://github.com/jupyterlab/jupyterlab/pull/13722/commits/ab8710e8252df3f7d545660538c46173c4831d4c for now

Reply:
Mostly because of the test failure of the previous commit, with the intent of dropping it later on but not necessarily in this PR to avoid blocking it.